### PR TITLE
remove trigger form fixes #2964

### LINKED ISSUE #2964: Remove `trigger` from rapid intake UI
`trigger` has been moved out of MVP1 so we should take it out of the UI.

### PR DESCRIPTION
**Pull request by tiftran**
Decided to smush #2941 into this PR as well

### CODE REVIEW

**Comment by pdehaan** [comment]
Does this need a migration to remove the field?
https://github.com/mozilla/experimenter/blob/5e5c11e93085aa99f427daf9e377a4dc366d1fa5/app/experimenter/experiments/migrations/0094_message_experiments.py#L47-L51

**Comment by tiftran** [comment]
> Does this need a migration to remove the field?
> 
> https://github.com/mozilla/experimenter/blob/5e5c11e93085aa99f427daf9e377a4dc366d1fa5/app/experimenter/experiments/migrations/0094_message_experiments.py#L47-L51

Based on the github issue description, it seems that trigger would isn't included in MVP1, but is still a field that we would like to have later? idk if that's correct. 
If that's the case, the trigger field isn't required, so it should not block workflows. To me, I don't see harm in leaving it there as we will need it later?, unless I'm missing something @pdehaan @jaredlockhart ?

**Comment by jaredlockhart** [comment]
> > Does this need a migration to remove the field?
> > https://github.com/mozilla/experimenter/blob/5e5c11e93085aa99f427daf9e377a4dc366d1fa5/app/experimenter/experiments/migrations/0094_message_experiments.py#L47-L51
> 
> Based on the github issue description, it seems that trigger would isn't included in MVP1, but is still a field that we would like to have later? idk if that's correct.
> If that's the case, the trigger field isn't required, so it should not block workflows. To me, I don't see harm in leaving it there as we will need it later?, unless I'm missing something @pdehaan @jaredlockhart ?

Yep that's exactly right @tiftran 👍

File: app/experimenter/static/rapid/components/forms/ExperimentForm.tsx [hunk 1]
```diff
@@ -163,10 +146,11 @@ const ExperimentForm: React.FC = () => {
           Is there a minimum Firefox Release version this experiment should be
           run on?
         </p>
-        <select
-          className="form-control w-100"
+        <XSelect
+          className="w-100"
           id="field-version"
           name="version"
+          options={[]}
```
Comment (jaredlockhart) [inline app/experimenter/static/rapid/components/forms/ExperimentForm.tsx:153]: Aha! Snuck that in there ;)

**Pull request merged**